config/libc/bionic.in

```diff
@@ -0,0 +1,185 @@
+# bionic options
+
+## depends on ! WINDOWS && ! BARE_METAL
+## depends on ARCH_arm || ARCH_mips || ARCH_x86
```

Comment:
Sorry I forgot about this. Sounds like this means all aarch64-*, arm*, mips*, and x86* samples need to be tweaked?